book.rb

```diff
@@ -0,0 +1,15 @@
+require './rental'
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
```

Comment:
- [ ] You have added a getter for `@rentals` which is good, but we also need a setter according to project requirements 😎